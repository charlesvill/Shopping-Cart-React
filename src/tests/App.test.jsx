import { describe, expect, it } from 'vitest';
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import NavBar from '../components/navbar/navbar.jsx';

describe("Navbar component", () => {
  it("renders NavBar", () => {
    const { container } = render(
      <BrowserRouter>
        <NavBar />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });

});

