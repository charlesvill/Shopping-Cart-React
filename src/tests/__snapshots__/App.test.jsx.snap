// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > renders NavBar 1`] = `
<div>
  <nav
    class="_nav_8210b0"
  >
    <div
      class="logoCont"
    >
      <a
        href="/"
      >
        <div
          class="domainName"
        >
          True North Games
        </div>
        <div
          class="domainLogo"
        />
      </a>
    </div>
    <div
      class="_searchCont_8210b0"
    >
      <form>
        <input
          placeholder="Search Games"
          type="text"
          value=""
        />
      </form>
    </div>
    <div
      class="cartCont"
    >
      <a
        href="/cart"
      >
        shopping cart
      </a>
    </div>
  </nav>
</div>
`;
